test/helpers/ndr_ui/bootstrap_helper_test.rb

```diff
@@ -183,6 +183,40 @@ class BootstrapHelperTest < ActionView::TestCase
       end
     end
 
+    test 'bootstrap_form_with' do
+      @output_buffer = bootstrap_form_with(
+        url: posts_path,
+        html: { id: 'preserve_me' }
+      ) do |form|
+        assert_kind_of BootstrapBuilder, form
+      end
+      assert_select 'form#preserve_me[autocomplete=off][action="/posts"]'
+
+      @output_buffer = bootstrap_form_with(
+        url: posts_path,
+        html: { id: 'preserve_me', class: 'form-inline' }
+      ) do |form|
+        assert_kind_of BootstrapBuilder, form
+      end
+      assert_select 'form#preserve_me.form-inline[autocomplete=off][action="/posts"]'
+
+      @output_buffer = bootstrap_form_with(
+        url: posts_path,
+        horizontal: true, html: { id: 'preserve_me' }
+      ) do |form|
+        assert_kind_of BootstrapBuilder, form
+      end
+      assert_select 'form#preserve_me.form-horizontal[autocomplete=off][action="/posts"]'
+    end
+
+    test 'bootstrap_form_with invalid autocomplete option' do
+      assert_raise RuntimeError do
+        bootstrap_form_with(url: posts_path, autocomplete: 'on') do |form|
+          assert_kind_of BootstrapBuilder, form
+        end
+      end
+    end
+
```

Comment:
Can we also have some tests to cover it working in `model: @post` mode (i.e. as `form_for`) as well as `url: posts_url` mode (aka `form_tag`) ?